### DISCUSSION TITLE
AutoImage Cant load gif ?

### DISCUSSION THREAD

**Discussion by oottoohh** [infinitered/ignite#2331] [Q&A]
do any one using this component to load gif ? i try by my self is not working or is it wrong to use it to load gif ?

**Comment by ryskin** [comment]
What platform do you try? https://reactnative.dev/docs/0.69/image#gif-and-webp-support-on-android
Would be good if you attach the image you try to use.

**Reply by oottoohh** [reply]
im not using expo, i already add dependecy fresco but when i try using AutoImage gif now show ..everything working fine if im just using image
